### PR TITLE
Start whodata thread only when whodata is configured

### LINKED ISSUE #4374: Whodata thread not starting (No rules added)
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.10.2-1 | Syscheck/whodata | Agent | Packages | RHEL 6.7 |

Hi team,

I'm having some problems configuring whodata to coexist with other audit rules.
My syscheck configuration is the following:
```xml
  <syscheck>
      <directories check_all="yes" realtime="yes" whodata="yes" recursion_level="0" restrict="^/etc/group$|^/etc/passwd$|^/etc/gshadow$|^/etc/shadow$">/etc/</directories>

      <whodata>
        <restart_audit>yes</restart_audit>
        <audit_key>time-change,identity,system-locale,MAC-policy,actions,fim_group,fim_passwd,fim_gshadow,fim_shadow</audit_key>
        <startup_healthcheck>no</startup_healthcheck>
       </whodata>

  </syscheck>
```
Audit rules at audit.rules:
```
# cat /etc/audit/rules.d/audit.rules
## First rule - delete all
-D

## Increase the buffers to survive stress events.
## Make this bigger for busy systems
-b 8192

## Set failure mode to syslog
#-f 1

# Feel free to add below this line. See auditctl man page
-w /etc/group -p wa -k fim_group
-w /etc/passwd -p wa -k fim_passwd
-w /etc/gshadow -p rwxa -k fim_gshadow
-w /etc/shadow -p rwxa -k fim_shadow
```

When the agent starts, I see these logs:
```
2019/12/16 09:29:06 ossec-syscheckd[17547] syscheck_audit.c:450 at audit_init(): INFO: (6026): Audit health check is disabled. Real-time Whodata could not work correctly.
2019/12/16 09:29:06 ossec-syscheckd[17547] audit_op.c:71 at audit_print_reply(): DEBUG: Audit rule loaded: -w /etc/group -p wa -k fim_group
2019/12/16 09:29:06 ossec-syscheckd[17547] audit_op.c:71 at audit_print_reply(): DEBUG: Audit rule loaded: -w /etc/passwd -p wa -k fim_passwd
2019/12/16 09:29:06 ossec-syscheckd[17547] audit_op.c:71 at audit_print_reply(): DEBUG: Audit rule loaded: -w /etc/gshadow -p rwxa -k fim_gshadow
2019/12/16 09:29:06 ossec-syscheckd[17547] audit_op.c:71 at audit_print_reply(): DEBUG: Audit rule loaded: -w /etc/shadow -p rwxa -k fim_shadow
2019/12/16 09:29:06 ossec-syscheckd[17547] syscheck_audit.c:458 at audit_init(): DEBUG: (6274): No rules added. Audit events reader thread will not start.
2019/12/16 09:29:06 ossec-syscheckd[17547] syscheck.c:479 at main(): WARNING: Audit events reader thread not started.
```

### PR DESCRIPTION
**Pull request by jesusjimsa**
| Related issue  | QA issue                        |
|---------------|----------------------- |
| #4374             | wazuh/wazuh-qa#695 |

## Description

This pull request fixes the case described in #4374 where, if an agent is added to two groups and the first one is configured with _whodata_ enabled but in the second one it is disabled, the _whodata_ thread would start even though there are no directories with _whodata_ enabled.

To fix this, the options configured for the directories are checked after reading the `agent.conf` file and the `syscheck.enable_whodata` is set to `1` only if there is at least one directory configured with _whodata_.

This pull request closes #4374.

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux
  - [x] Windows
  - [x] MAC OS X
- [x] Source installation
- [ ] Package installation
- [x] Source upgrade
- [ ] Package upgrade
- [x] Review logs syntax and correct language
- [x] QA templates contemplate the added capabilities

- Memory tests for Linux
  - [x] Scan-build report
  - [ ] Coverity
  - [x] Valgrind (memcheck and descriptor leaks check)
  - [x] Dr. Memory
  - [x] AddressSanitizer

### CODE REVIEW

**Comment by jesusjimsa** [comment]
test

**Comment by jesusjimsa** [comment]
## scan-build
No bugs found.

**Comment by jesusjimsa** [comment]
## Valgrind
No leaks found.
```
==26625== LEAK SUMMARY:
==26625==    definitely lost: 0 bytes in 0 blocks
==26625==    indirectly lost: 0 bytes in 0 blocks
==26625==      possibly lost: 0 bytes in 0 blocks
==26625==    still reachable: 11,089 bytes in 41 blocks
==26625==         suppressed: 0 bytes in 0 blocks
==26625== Reachable blocks (those to which a pointer was found) are not shown.
==26625== To see them, rerun with: --leak-check=full --show-leak-kinds=all
```

**Comment by jesusjimsa** [comment]
## AddressSanitizer
No bugs found.

**Comment by jesusjimsa** [comment]
## Dr. Memory
No errors found.
```
~~31691~~
~~31691~~ NO ERRORS FOUND:
~~31691~~       0 unique,     0 total unaddressable access(es)
~~31691~~       0 unique,     0 total uninitialized access(es)
~~31691~~       0 unique,     0 total invalid heap argument(s)
~~31691~~       0 unique,     0 total warning(s)
~~31691~~       0 unique,     0 total,      0 byte(s) of leak(s)
~~31691~~       0 unique,     0 total,      0 byte(s) of possible leak(s)
~~31691~~ ERRORS IGNORED:
~~31691~~      70 unique,   138 total,  30358 byte(s) of still-reachable allocation(s)
~~31691~~          (re-run with "-show_reachable" for details)
```

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 352913ee8588999e7e83ed4d9bd01c7d2a2f9894
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1630/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1630/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1630/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1630.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1630/Analysis_dynamic_valgrind-v3.13.0-B1630.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 352913ee8588999e7e83ed4d9bd01c7d2a2f9894
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1631/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1631/ossec.log)
**Coverage:** 

```json
Lines executed:15.64% of 3056

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1631/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1631.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1631/Analysis_dynamic_valgrind-v3.13.0-B1631.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 352913ee8588999e7e83ed4d9bd01c7d2a2f9894
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1632/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1632/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1632/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1632.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1632/Analysis_dynamic_valgrind-v3.13.0-B1632.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 352913ee8588999e7e83ed4d9bd01c7d2a2f9894
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1633/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1633/ossec.log)
**Coverage:** 

```json
Lines executed:15.64% of 3056

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1633/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1633.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1633/Analysis_dynamic_valgrind-v3.13.0-B1633.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1636/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1636/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1636/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1636.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1636/Analysis_dynamic_valgrind-v3.13.0-B1636.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1637/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1637/ossec.log)
**Coverage:** 

```json
Lines executed:26.67% of 3063

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1637/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1637.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1637/Analysis_dynamic_valgrind-v3.13.0-B1637.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1662/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1662/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1662/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1662.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1662/Analysis_dynamic_valgrind-v3.13.0-B1662.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1663/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1663/ossec.log)
**Coverage:** 

```json
Lines executed:26.67% of 3063

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1663/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1663.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1663/Analysis_dynamic_valgrind-v3.13.0-B1663.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1664/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1664/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1664/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1664.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1664/Analysis_dynamic_valgrind-v3.13.0-B1664.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1665/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1665/ossec.log)
**Coverage:** 

```json
Lines executed:26.64% of 3063

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1665/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1665.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1665/Analysis_dynamic_valgrind-v3.13.0-B1665.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1688/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1688/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1688/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1688.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1688/Analysis_dynamic_valgrind-v3.13.0-B1688.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1689/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1689/ossec.log)
**Coverage:** 

```json
Lines executed:26.67% of 3063

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1689/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1689.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1689/Analysis_dynamic_valgrind-v3.13.0-B1689.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1692/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1692/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1692/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1692.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1692/Analysis_dynamic_valgrind-v3.13.0-B1692.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1694/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1694/ossec.log)
**Coverage:** 

```json
Lines executed:26.67% of 3063

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1694/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1694.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1694/Analysis_dynamic_valgrind-v3.13.0-B1694.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1804/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1804/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1804/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1804.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1804/Analysis_dynamic_valgrind-v3.13.0-B1804.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 63ec77851e2db0be106cb2eb11497244661f04fb
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1803/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1803/ossec.log)
**Coverage:** 

```json
Lines executed:26.67% of 3063

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1803/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1803.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1803/Analysis_dynamic_valgrind-v3.13.0-B1803.log)

File: src/syscheckd/config.c [hunk 1]
```diff
@@ -80,6 +80,21 @@ int Read_Syscheck_Config(const char *cfgfile)
     ReadConfig(modules, AGENTCONFIG, &syscheck, NULL);
 #endif
 
+    int it = 0;
+
+    /* Check directories options to determine whether to start the whodata thread or not */
+    if (syscheck.dir) {
```
Comment (bah07) [inline src/syscheckd/config.c:86]: I think this code is redundant. When FIM starts, a series of messages are printed informing about the configuration of the directories. This check should be done in `src/syscheckd/main.c:234`.

Comment (jesusjimsa) [reply src/syscheckd/config.c]: Done 186bb0b

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 186bb0b355ef44ed944d25691f6e6f96948f7766
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1837/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1837/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1837/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1837.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1837/Analysis_dynamic_valgrind-v3.13.0-B1837.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 186bb0b355ef44ed944d25691f6e6f96948f7766
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1838/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1838/ossec.log)
**Coverage:** 

```json
Lines executed:26.78% of 3189

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1838/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1838.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1838/Analysis_dynamic_valgrind-v3.13.0-B1838.log)

**Comment by jesusjimsa** [comment]
Integration tests completed successfully:
* [agent_centos_html_report_test_integration_B149_20200515125558.zip](https://github.com/wazuh/wazuh/files/4634423/agent_centos_html_report_test_integration_B149_20200515125558.zip)
* [agent_ubuntu_html_report_test_integration_B149_20200515125558.zip](https://github.com/wazuh/wazuh/files/4634424/agent_ubuntu_html_report_test_integration_B149_20200515125558.zip)
* [agent_windows_html_report_test_integration_B149_20200515125558.zip](https://github.com/wazuh/wazuh/files/4634425/agent_windows_html_report_test_integration_B149_20200515125558.zip)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 186bb0b355ef44ed944d25691f6e6f96948f7766
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1844/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1844/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1844/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1844.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1844/Analysis_dynamic_valgrind-v3.13.0-B1844.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 186bb0b355ef44ed944d25691f6e6f96948f7766
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1845/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1845/ossec.log)
**Coverage:** 

```json
Lines executed:26.78% of 3189

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1845/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1845.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1845/Analysis_dynamic_valgrind-v3.13.0-B1845.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind logcollector analysis:
**Commit:** 186bb0b355ef44ed944d25691f6e6f96948f7766
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1867/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1867/ossec.log)
**Coverage:** 

```json
Lines executed:31.41% of 1859

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1867/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1867.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1867/Analysis_dynamic_valgrind-v3.13.0-B1867.log)

**Comment by wazuhci** [comment]
## Analysis_dynamic_valgrind syscheckd analysis:
**Commit:** 186bb0b355ef44ed944d25691f6e6f96948f7766
**Parameters:** track-origins="yes" leak-check="full" keep-stacktraces="alloc-and-free" track-fds="yes" num-callers="20" show-leak-kinds="definite,indirect"
**Agent config:** [ossec.conf](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1868/ossec.conf)
**Agent logs:** [ossec.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1868/ossec.log)
**Coverage:** 

```json
Lines executed:26.75% of 3189

```
**Coverage report detail:** [coverage-report-summary](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1868/coverage-report-detail)
**Valgrind result:** [Analysis_dynamic_valgrind-v3.13.0-B1868.log](https://ci.wazuh.com/data/Analysis_dynamic_valgrind/v3.13.0/B1868/Analysis_dynamic_valgrind-v3.13.0-B1868.log)

**Pull request merged**